pkg/accounting/accounting_test.go

```diff
@@ -102,7 +101,7 @@ func TestMutex(t *testing.T) {
 
 // TestAccountingAddBalance does several accounting actions and verifies the balance after each steep
 func TestAccountingAddBalance(t *testing.T) {
-	logger := logging.New(io.Discard, 0)
```

Comment:
why the need to change this? the new logger should continue to work with `io.Discard`.

Reply:
This is just a temporary change and will be reverted when the migration is completed.